conanfile.py

```diff
@@ -105,92 +104,98 @@ class LibnameConan(ConanFile):
         "zlib/1.2.11",
     )
 
+    # TODO - Packages required but not listed in this recipe
+    # wayland-scanner (libwayland-dev on Ubuntu)
```

Comment:
currently, the recipe does not build the wayland support, this is why these are not needed (yet). I'd have to take a look if creating a conan package for wayland is worth considering (that's why I let this comment https://github.com/bincrafters/conan-mesa/pull/1/files#diff-2b1d42f71f22b7ea0412d7602dec166fR164)

Reply:
When I tried to buying with conanio/gcc5, the build broke and said wayland was required. Is there any way to avoid it ?

Reply:
this was because of the typo in here: 8b09ee5#diff-2b1d42f71f22b7ea0412d7602dec166fR292
It should not try to build wayland support if platforms option is properly passed